src/ingest-pipeline/airflow/dags/salmon_rnaseq.py

```diff
@@ -30,11 +30,11 @@
     make_send_status_msg_function,
     get_tmp_dir_path,
     pythonop_get_dataset_state,
+    HMDAG,
+    get_queue_resource,
+    get_threads_resource,
 )
 
-# to be used by the CWL worker
-THREADS = 6
-
 
 def generate_salmon_rnaseq_dag(params: SequencingDagParameters) -> DAG:
```

Comment:
I'm not sure how to add a comment to the `salmon-rnaseq` submodule update, so I'm adding it here -- is that spurious? The GitHub diff view doesn't show any relevant changes.

Reply:
I apparently updated salmon-rnaseq to match the current version on devel to run a test.  That was different from the version in effect when the branch was created, so it showed up as a change when I created the PR and I failed to notice.  I've cleaned it up; devel already has the change in place.